### ISSUE TITLE
Sending link to Yaacc with share button fails most of the time.

### ISSUE THREAD

**Issue by Mahava** [tobexyz/yaacc-code#82]
Im trying to play an url from a phone app using the "share" button to send it to yaacc.   I want to play it on my computer.  Sometimes it works, sometimes not.    Can you tell me how to set this up so yaacc works consistently?

-I send a media link from UNCHAINED app by pressing SHARE then selecting YAACC
-The renderer is POTPLAYER on PC using MACAST as receiver
-YAACC opens then sometimes closes immediately.  When it does stay open, I SHARE the link again.  Ocassionally it plays on POTPLAYER but most often YAACC closes or I get a message that the app keeps stopping.

What am I doing wrong?

Ive attached YAACC's log:

[log.txt](https://github.com/tobexyz/yaacc-code/files/12613027/log.txt)

**Comment by tobexyz** [comment]
Hi @Mahava,
thank you for your report. The log was really helpful! I tried a fix by running the commands where the exception occurs on the ui thread now. Please give it a try by testing the branch [feat/issue82](https://github.com/tobexyz/yaacc-code/tree/feat/issue82) if you are able to.

**Comment by Mahava** [comment]
Thans kindly.   My home network is down.. Once its back up I will try it and report back.

**Comment by Mahava** [comment]
Hi,

I was able to build issue 82 debug apk and install it.   It works great for my needs now.  The biggest issue I had is the gradlew build kept failing both in cmd window and inside terminal window of Android Studio despite Android home being set..  What worked is using AS menu and selecting build>build bundles>build apk.  AS also recommended that i upgrade the project which i did not do.

Another minor thing is that a new yaacc window opens each time i share a link to it.  BTW, none of the boxes in my server settings section are checked.  I looked at the error log and it was empty.   If youd like a debug log, let me know.
